test/test_torch.py

```diff
@@ -12784,6 +12784,61 @@ def test_fill_diagonal(self):
         e1.fill_diagonal_(v, wrap=True)
         self.assertEqual(e1, e2)
 
+    def test_sign(self):
+
+        # Check sign on all supported backends
```

Comment:
We are using this pattern all over the place, they are self explanatory and don't need to be commended (we can save a lot of vertical space by removing new lines and this comments).